server/app/router.js

```diff
@@ -56,6 +56,16 @@ router.get(
   authenticateUser,
   transaction.readTransactionById
 );
+router.get(
+  "/transactionsamount/:periodFilter",
+  authenticateUser,
+  transaction.expensesAmount
+);
+router.get(
+  "/transactionscategoryamount/:periodFilter",
```

Comment:
**issue**: That name is even weirder to me hahah

Reply:
Yeah i didn't had a good name idea ahah.
What it returns is, for each category, according to a specific period, the sum of all the categories transactions.
And of course categoryless transactions are taken into account thanks to the LEFT JOIN of the query.